content/preferences-settings/shortcuts.md

```diff
@@ -5,133 +5,204 @@ weight: 120
 draft: false
 ---
 
-Much of the functionality of darktable can be accessed via shortcuts using the keyboard or keyboard/mouse combinations. These shortcuts are user-configurable via the _shortcuts_ tab. 
+You can perform almost any action in darktable with a keyboard/mouse shortcut. You can also use various other input devices, including MIDI devices and game controllers -- see the [midi device support](../special-topics/midi-device-support.md) section for details.
 
-Many important shortcut actions are provided with default key combinations, but most must be manually configured by the user. Any key may be used for a keyboard shortcut, and may be combined with the Shift, Control or Alt modifier keys (or any combination thereof).
+# defining shortcuts
 
-When you open the _shortcuts_ tab you are initially presented with a hierarchical list of all actions that can be applied with a keyboard shortcut. At the top of this hierarchy is a short list of key [_categories_](#shortcut-categories) defined below.
+A _shortcut_ is a combination of key or button presses and/or mouse or device movements that performs an _action_ in darktable.
 
-# add or amend a shortcut
+A single action may have multiple shortcuts but a single shortcut can only be linked to one action in a given darktable view -- you can't chain actions together except by applying a preset or style. You can, however, set up a single shortcut that does one thing in the lighttable view, say, and another in the darkroom view.
 
-In order to add or amend a shortcut, first navigate to the action you want to change and double-click on it. You will be prompted to press the new key combination to be mapped to the selected action. 
+## simple shortcuts
 
-If a conflict is found you will be given the option to retain the existing shortcut or replace it. Depending on context, it is possible to use the same keyboard shortcut for multiple actions. For example the same key combination may be used for one action in the lighttable view and another in the darkroom view.
+All shortcuts involving the keyboard/mouse must be initiated with a key combination on the keyboard. This combination must consist of at least one key press, plus an optional modifier (Shift, Ctrl or Alt, either alone or in combination). You can use a single-, double- or triple-key-presses and the final key-press may be lengthened to provide an additional option. You cannot press any other button or key after a long-press, and you cannot extend it (see below) -- the release of the final key must end the shortcut.
```

Comment:
"A long-press is triggered when the key is held longer than the double-click period and then released without any other presses or moves." Since the key has now been released, no further combinations including it can be generated. It reads a bit confusing here; maybe only mention this under "extending simple shortcuts"? "Since a long press (or click) shortcut ends with the key (or mouse button)being released, it cannot be extended".